.circleci/config.yml

```
@@ -282,10 +329,25 @@ workflows:
           image_tag: heroku/pack:20
           image_tag_alias: heroku/pack:20
           requires:
+            - test-go-20
+            - test-java-20
+            - test-php-20
+            - test-python-20
             - create-20-run-image
```

Similarly, this dependency is redundant (or if we want to keep it, we should add a similar one to the 18 equivalent)
```suggestion
            - create-20-run-image
```